timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/v5/graphql/mutations/SummaryPropsMutationDataFetcher.java

```diff
@@ -34,20 +38,26 @@ public Object get(DataFetchingEnvironment env) {
     String collectionUri = env.getArgument("collectionUri");
     Map viewConfig = env.getArgument("summaryProperties");
 
+    ContextData contextData = env.getContext();
+    UserPermissionCheck userPermissionCheck = contextData.getUserPermissionCheck();
+
     Tuple<String, String> userAndDataSet = PromotedDataSet.splitCombinedId(dataSetId);
 
     String ownerId = userAndDataSet.getLeft();
     String dataSetName = userAndDataSet.getRight();
-    if (dataSetRepository.dataSetExists(ownerId, dataSetName)) {
-      DataSet dataSet = dataSetRepository.getDataSet(ownerId, dataSetName).get();
-      dataSet.getQuadStore();
+    Optional<DataSet> dataSet = dataSetRepository.getDataSet(userAndDataSet.getLeft(),dataSetId);
```

Comment:
I think this will use the ownerId as the userId and will try to split the dataSetId.

I think it's best if we remove the overload from dataSetRepository and force splitting on the calling side